ts/private/ts_project_worker.js

```diff
@@ -467,11 +467,6 @@ function createFilesystemTree(root, inputs) {
 
 
 /** Program and Caching */
-function isExternalLib(path) {
-    return path.includes('external') &&
```

Comment:
This condition fails if using a custom version of TypeScript. Updated the logic in this file with a different way of checking this.